templates/blog/list.html

```diff
@@ -46,14 +50,24 @@
         <ul>
             <li id="blog-tab" class="tab active"><a href="#">
                 <i class="tab-icon fa fa-info-circle"></i> {{ _('Blog') }}
-            </a></li>
+            </a></li>tạp chí
```

Comment:
wut is this

Reply:
They are redundant words, so I removed them.